sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorClient.cs

```diff
@@ -342,6 +342,338 @@ public virtual Response<IReadOnlyList<MetricSeriesData>> GetMetricSeriesData(str
+        }
+
+        /// <summary>
+        ///
+        /// </summary>
+        /// <param name="options"></param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        public virtual AsyncPageable<MetricFeedback> GetMetricFeedbacksAsync(GetMetricFeedbackOptions options, CancellationToken cancellationToken = default)
+        {
+            Argument.AssertNotNull(options, nameof(options));
+
+            MetricFeedbackFilter queryOptions = new MetricFeedbackFilter(options.MetricId)
+            {
+                DimensionFilter = options.DimensionFilter,
+                EndTime = options.EndTime,
+                FeedbackType = options.FeedbackType,
+                StartTime = options.StartTime,
+                TimeMode = options.TimeMode
+            };
+            int? skip = options?.SkipCount;
+            int? top = options?.TopCount;
+
+            async Task<Page<MetricFeedback>> FirstPageFunc(int? pageSizeHint)
+            {
+                using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(MetricsAdvisorClient)}.{nameof(GetMetricFeedbacks)}");
+                scope.Start();
+
+                try
+                {
+                    Response<MetricFeedbackList> response = await _serviceRestClient.ListMetricFeedbacksAsync(queryOptions, skip, top, cancellationToken).ConfigureAwait(false);
+                    return Page.FromValues(response.Value.Value, response.Value.NextLink, response.GetRawResponse());
+                }
+                catch (Exception e)
+                {
+                    scope.Failed(e);
+                    throw;
+                }
+            }
+
+            async Task<Page<MetricFeedback>> NextPageFunc(string nextLink, int? pageSizeHint)
+            {
+                using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(MetricsAdvisorClient)}.{nameof(GetMetricFeedbacks)}");
+                scope.Start();
+
+                try
+                {
+                    Response<MetricFeedbackList> response = await _serviceRestClient.ListMetricFeedbacksNextAsync(nextLink, queryOptions, cancellationToken).ConfigureAwait(false);
+                    return Page.FromValues(response.Value.Value, response.Value.NextLink, response.GetRawResponse());
+                }
+                catch (Exception e)
+                {
+                    scope.Failed(e);
+                    throw;
+                }
+            }
+
+            return PageableHelpers.CreateAsyncEnumerable(FirstPageFunc, NextPageFunc);
+        }
+
+        /// <summary>
+        ///
+        /// </summary>
+        /// <param name="options"></param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        public virtual Pageable<MetricFeedback> GetMetricFeedbacks(GetMetricFeedbackOptions options, CancellationToken cancellationToken = default)
```

Comment:
I could go either way - Java has `listMetricFeedback` , so I think I'll change it to singular `GetMetricFeedback`.

Reply:
`GetMetricFeedback` is already used for retrieving a single element, so we would have them as overloads. I think the difference between the parameters won't convey this idea to users properly.

I just asked about the name because I wasn't sure if it could sound off to English speakers, but it doesn't really feel weird to me. If you think it's okay, I'm in favor of keeping it as `Feedbacks`.